bahmni-commons-api/src/main/java/org/bahmni/module/bahmnicommons/contract/patient/search/PatientIdentifierQueryHelper.java

```diff
@@ -25,7 +25,7 @@ public String appendToJoinClause(String join) {
 				" JOIN patient_identifier_type pit ON pi.identifier_type = pit.patient_identifier_type_id AND pi.voided IS FALSE " +
 				" JOIN global_property gp ON gp.property IN ('bahmni.primaryIdentifierType' "+extraIdentifierQuery+")" +
 				" AND gp.property_value LIKE concat('%', pit.uuid, '%')" +
-				" AND pi.identifier LIKE '%" +StringEscapeUtils.escapeSql(identifier)+ "%' GROUP BY pi.patient_id) " +
+				" AND pi.identifier LIKE '%" + SqlQueryHelper.escapeSQL(identifier, true, null) + "%' GROUP BY pi.patient_id) " +
```

Comment:
Same comment as above. PatientIdentifierQueryHelper is only used in call chain of deprecated method, which is not used any more.